src/mosaic.mjs

```
@@ -176,88 +196,95 @@
-      tile,
-      meta: metadataByUuid[rows[index].uuid],
-    }))
-    .filter(({ meta, tile }, index) => {
-      if (!meta) {
-        console.warn(`Null metadata found for tile at index ${index}, skipping...`);
-        return false;
-      }
-      return true;
+  // Resolve all row-based tile promises
+  const resolvedRowTiles = await Promise.all(rowTiles.map((rt) => rt.promise));
+  let finalTiles = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'finalTiles' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)